contracts/modules/cover/Cover.sol

```diff
@@ -0,0 +1,576 @@
+
+import "@openzeppelin/contracts-v4/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
+import "../../interfaces/ICover.sol";
+import "../../interfaces/IStakingPool.sol";
+import "../../interfaces/IPool.sol";
+import "../../abstract/MasterAwareV2.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "../../interfaces/ICoverNFT.sol";
+import "hardhat/console.sol";
+
+contract Cover is ICover, MasterAwareV2 {
+
+  uint public capacityFactor;
+  uint public coverCount;
+
```

Comment:
Make `capacityFactor` and `coverCount` uint32 and move them near `coverNFT` address so they can use the same slot (32+32+160 = 224).